src/main/java/bio/terra/service/auth/iam/sam/SamIam.java

```
@@ -751,6 +752,16 @@ private String signUrlForBlobInner(
     return samApiService.googleApi(userReq.getToken()).getRequesterPaysSignedUrlForBlob(request);
   }
 
+  public void azureCreateManagedResourceGroup(
+      AuthenticatedUserRequest userReq,
+      String billingProfileId,
+      ManagedResourceGroupCoordinates managedResourceGroupCoordinates)
+      throws ApiException {
+    samApiService
```

The pattern we follow in `SamIam` public methods is to include automatic retry capability of a private inner method that actually calls the Sam API.  This is for added resilience against Sam being flaky, and we favor the resilience over failing fast in all cases.  Can you follow that in this new method too?

(Optional extra context) here's a past PR where I worked on modifications to `SamRetry` which may be a helpful concrete example of its behavior: https://github.com/DataBiosphere/jade-data-repo/pull/1410